configs/.env.development.local

```diff
@@ -0,0 +1,22 @@
+ENVIRONMENT_NAME=development
```

Comment:
Why remove existing approach only to replace it with pretty much the same? 
Better approach would be build on it, improve it. The server config package already reads `.env` file.

Reply:
1. Approached this task as making gradual improvement. I could have rewritten much in the config package, but wanted to start small with just getting server running locally (which isn't documented right now) by having establishing a local development environment template in /configs, that's self-documenting via .env and scripts.

2. I also didn't want to have a large set of changes just starting out in contribution without first establishing the server running successfully through already defined inputs.

3. This PR also self documents getting the server running in local development, which now allows for future work to build on with tests that can reference established environment input conditions.

4. @kevinmgates is currently working on issue #18 that is going to allow for an environment name to be set so that we can identify DEVELOPMENT and seed a database, allowing the Python stub API to be removed. Because he's working in config package, I didn't want to make a merge difficult for his work in the meantime.

Reply:
My earlier point was that we already use a package that will read all environment variables from a file named `.env`. So instead of re-implementing that by exporting variables from another file, let's stick them into `.env` file and that's it.

Reply:
Here is the link to the documentation of that package: https://github.com/elsevierlabs-os/go-envs#how-it-works

Reply:
Updated.